app/ts/components/Transactions.tsx

```diff
@@ -72,13 +70,13 @@ export const Transactions = ({
 				uniqueAddresses.map((address) =>
 					fetch(
 						`https://api${appSettings.peek().relayEndpoint === MEV_RELAY_GOERLI ? '-goerli' : ''
-						}.etherscan.io/api?module=contract&action=getabi&address=${utils.getAddress(address.toLowerCase())}&apiKey=PSW8C433Q667DVEX5BCRMGNAH9FSGFZ7Q8`,
+						}.etherscan.io/api?module=contract&action=getabi&address=${getAddress(address.toLowerCase())}&apiKey=PSW8C433Q667DVEX5BCRMGNAH9FSGFZ7Q8`,
```

Comment:
API Key embedded in websites rarely ends well.  I'm guessing this is unavoidable?

Reply:
Yeah, its also taken from ABI Ninja